lib/internal/url.js

```diff
@@ -315,6 +317,22 @@ function onParseHashComplete(flags, protocol, username, password,
   this[context].fragment = fragment;
 }
 
+function special(self) {
+  return (self[context].flags & URL_FLAGS_SPECIAL) !== 0;
+}
+
+function cannotBeBase(self) {
+  return (self[context].flags & URL_FLAGS_CANNOT_BE_BASE) !== 0;
+}
+
+// https://url.spec.whatwg.org/#cannot-have-a-username-password-port
+function cannotHaveUsernamePasswordPort(self) {
+  const { host, scheme } = self[context];
+  return (host == null || host === '') ||
+         cannotBeBase(self) ||
+         scheme === 'file:';
+}
```

Comment:
Similar to the above … why this change? It really helps with readability to have them inline and they don’t show up in inspection…

Reply:
Yepp, reverting.